src/app/video-games/detail/components/button-toggle/button-toggle.component.scss

```diff
@@ -0,0 +1,31 @@
+button {
+  padding: 0.6rem 1.25rem;
+  border-radius: 0.35rem;
+  border: none;
+  font-size: 1.2em;
+  margin-left: 0.4rem;
+  margin-right: 0.4rem;
+  color: #FFF;
+  background: #F44336;
+  box-shadow: 0px 5px 0px 0px #D32F2F;
+
+  &:active {
+    box-shadow: 0px 2px 0px 0px #D32F2F;
+    transform: translateY(3px);
+  }
+
+  &.active {
```

Comment:
I think some of the style definitions over the ones above this. Can we maybe clean this up? :)